jsapp/js/components/account/dataStorage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Let's start naming route components as `somethingSomethingRoute.tsx` 👍 Unless it's not a route component?